docs/tasks.md

```diff
@@ -0,0 +1,20 @@
+Tasks
+=====
+
+Tasks enabled file-centric workflows in Box. User can create tasks on files and assign them to collaborators on Box.
```

Comment:
I've fixed that, but this needs to be fixed i[n the official API ](https://docs.box.com/reference#task-object-1)reference when I got this from.